### ISSUE TITLE
Optifine Is Outdated

### ISSUE THREAD

**Issue by AgentCPU0** [sp614x/optifine#7667]
The title says it all, but I'm going to explain anyway

Optifine was great for older versions of the game, and if you only plan to play vanilla, it works fine on modern versions, as well. But there's a massive issue that still plagues Optifine to this day, and that's mods

It's no secret that mods drastically change the game, and it's one of the things that makes the game more enjoyable for many players. Older versions of the game ran Optifine with little to no issues, while later versions had a number of hiccups. This is due to Optifine using outdated code, and trying to render things that the game no longer uses causes issues

What makes it worse, even vanilla isn't safe from issues with Optifine. VanillaTweaks, a customizable resource pack that adjusts the game without ruining the vanilla feel, has warnings on some of the options stating that having Optifine has potential to not work. So, if neither vanilla nor modded works with Optifine, then it has to be cause by Optifine itself, right?

The thing that makes this even more frustrating is the devs don't want to make it open source. They don't want players making and distributing their own copies of Optifine without permission, which makes sense. But with it being so broken and so outdated, that risk would be more beneficial than forcing players to use a buggy, outdated, incompatible mess

And it gets even worse for Optifine, as we now have competitors. Mods like Soduim, Lithium, Iris Shaders, and Dynamic FPS, all of which aim to do things Optifine has done for years, but improved for the more recent versions of the game. In some cases, they work better than Optifine, which says a lot more about Optifine itself than it does about those mods

In conclusion, if you're just a player, don't use Optifine. You have better options now. You just need to filter mods by performance when searching for them. And to the devs, please, stop ignoring us. Either make it open source, and focus on releasing the base versions, or ask for outside help. There is absolutely no shame in doing either

**Comment by UsernameGeri** [comment]
You definitely seem to care a lot about Optifine, having typed this whole wall up. But as you said, it's time to let go of it. Updates take ages to come out, practically zero communication from a single dev, and there are tons of alternatives for all of Optifine's features. This mod is a relic of a bygone era, stop clinging to it.

**Comment by AgentCPU0** [comment]
This isn't whether or not I care about Optifine. It about trying to get the devs to understand that the mod is outdated at best, and completely useless at worst

**Comment by UsernameGeri** [comment]
I'm sure the dev understands lol.

**Comment by IMS212** [comment]
literally not a bug

**Comment by Jiingy** [comment]
This is not a valid report. Please note that the "developers" of OptiFine is only one person.
